packages/enzyme/package.json

```
@@ -39,7 +39,7 @@
   "license": "MIT",
   "dependencies": {
     "array.prototype.flat": "^1.2.4",
-    "cheerio": "=1.0.0-rc.3",
+    "cheerio": "=1.0.0-rc.11",
```

```suggestion
    "cheerio": "^1.0.0-rc.11",
```
i suspect this will break tests, which is why it's pinned to rc3.